static/app/views/dashboardsV2/gridLayout/sortableWidget.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+
+import IssueWidgetCard from '../issueWidgetCard';
+import {Widget, WidgetType} from '../types';
+
+import WidgetCard from './widgetCard';
+
+type Props = {
+  widget: Widget;
+  dragId: string;
+  isEditing: boolean;
+  onDelete: () => void;
+  onEdit: () => void;
+};
+
+function SortableWidget(props: Props) {
+  const {widget, isEditing, onDelete, onEdit} = props;
+
+  const widgetProps = {
+    widget,
+    isEditing,
+    onDelete,
+    onEdit,
+    showContextMenu: true,
+    // TODO(nar): These aren't necessary for supporting RGL
```

Comment:
I'm thinking of removing the dependencies on these props later, similar to how I would for addressing unnecessary props in `SortableWidget`